### PR TITLE
update silly wisher assets to 0.33

### PR DESCRIPTION
**Pull request by furrycatherder**
## Describe your changes

Updates to the latest Silly Wisher assets and adds a script for processing the files extracted from the APK.

## Issue or discord link

https://discord.com/channels/785153694478893126/819643218446254100/1189263910914756720

## Checklist before requesting a review (leave this PR as draft if any part of this list is not done.)

- [x] I have commented my code, in hard-to understand areas.
- [x] I have made corresponding changes to README or wiki.
- [ ] For front-end changes, I have updated the corresponding English translations.
- [x] Ran `yarn run mini-ci` locally to validate format + lint.
- [ ] If there were format issues, I ran `nx format write` to resolve them automatically.

## TBD

- [x] Convert `parse_i2languages.py` to JavaScript
- [x] Add more detail about APK extraction
- [ ] ~~Use static image optimization or external CDN for image assets~~

### CODE REVIEW

**Comment by furrycatherder** [comment]
Closes #1337

File: libs/silly-wisher/README.md [hunk 1]
```diff
@@ -3,3 +3,10 @@
 This library was generated with [Nx](https://nx.dev).
 
 This is a library for the Silly Wisher Assets used by Genshin Optimizer, with their permission to use for the April fools event.
+
+## Updating the splash and portrait art
+
+You can extract the assets from the Silly Wisher APK. I used
```
Comment (nguyentvan7) [inline libs/silly-wisher/README.md:9]: I believe this is the way I was experimenting with it as well. Could you document any commands you needed to run, and the location of the assets once extracted, etc?

Also note down that images will need to be resized, and to what resolution

File: libs/silly-wisher-names/Translated/parse_i2languages.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import csv
```
Comment (nguyentvan7) [inline libs/silly-wisher-names/Translated/parse_i2languages.py:1]: Possibly should migrate to TS

and add a way to run this via nx

**Comment by frzyc** [comment]
I think it also might be a good idea to find a place (like README) to show which sillywisher version the assets were extracted, so we can more easily tell if it needs to be updated.

**Comment by frzyc** [comment]
> Use static image optimization or external CDN for image assets

I'd say we can try to avoid doing this for now, since that create extra dependency and complexity that we are probably not staffed to manage.

File: libs/silly-wisher/VERSION [hunk 3]
```diff
@@ -0,0 +1 @@
+0.33
```
Comment (nguyentvan7) [inline libs/silly-wisher/VERSION:1]: What is the purpose of this?

File: libs/silly-wisher/README.md [hunk 4]
```diff
@@ -3,3 +3,27 @@
...
+```bash
+mkdir silly-wisher
+unzip -d silly-wisher "Silly Wisher_${SILLY_WISHER_VERSION}_apkcombo.com.apk"
+
+mkdir asset-ripper
+unzip -d asset-ripper AssetRipper.zip
+./asset-ripper/AssetRipper silly-wisher
+```
+
+Now the splash and portrait files should be in
```
Comment (nguyentvan7) [inline libs/silly-wisher/README.md:26]: nit: please add the current image sizes we use here (256x256 for portrait, 512x512 for splash)

File: libs/silly-wisher/VERSION (hunk 3 above)
Comment (frzyc) [reply libs/silly-wisher/VERSION:1]: I requested it:
> I think it also might be a good idea to find a place (like README) to show which sillywisher version the assets were extracted, so we can more easily tell if it needs to be updated.

**Comment by nguyentvan7** [comment]
https://genshin-optimizer-prs.github.io/pr/1396/frontend

**Review by frzyc — APPROVED** [review]

**Pull request merged**